tests/test_session.py

```
@@ -0,0 +1,116 @@
+from cross_auth._session import (
+    SessionConfig,
+    authenticate,
+    create_session,
+    delete_session,
+    get_session,
+    make_clear_cookie,
+    make_session_cookie,
+)
+from cross_auth._storage import AccountsStorage, SecondaryStorage
+
+TEST_PASSWORD = "password123"
```

Ruff (0.14.14)

[error] Possible hardcoded password assigned to: "TEST_PASSWORD"

(S105)